src/node_binding.cc

```diff
@@ -685,8 +685,7 @@ void GetLinkedBinding(const FunctionCallbackInfo<Value>& args) {
 
   Local<Object> module = Object::New(env->isolate());
   Local<Object> exports = Object::New(env->isolate());
-  Local<String> exports_prop =
-      String::NewFromUtf8Literal(env->isolate(), "exports");
+  Local<String> exports_prop = FIXED_ONE_BYTE_STRING(env->isolate(), "exports");
```

Comment:
If applicable:

```suggestion
  Local<String> exports_prop = FIXED_ONE_BYTE_STRING(env->isolate(), env->exports_string());
```

Reply:
If there is already an `env->exports_string()` then it would be `Local<String> exports_prop = env->exports_string()` ... and yes, looks like there is!
```suggestion
  Local<String> exports_prop = env->exports_string();`
```

Reply:
ah yeah that's true. I didn't read the whole context